src/RageTimer.cpp

```diff
@@ -35,11 +35,11 @@ const std::int64_t ONE_SECOND_IN_MICROSECONDS_LL = 1000000LL;
 const double ONE_SECOND_IN_MICROSECONDS_DBL = 1000000.0;
 
 const RageTimer RageZeroTimer(0,0);
-static std::uint64_t g_iStartTime = ArchHooks::GetMicrosecondsSinceStart( true );
```

Comment:
Since the other PR has `GetTimeSinceStartSeconds`, I think renaming to `GetTimeSinceStartMicroseconds` is in line.

Reply:
> Since the other PR has `GetTimeSinceStartSeconds`, I think renaming to `GetTimeSinceStartMicroseconds` is in line.

The one concern I have is that `GetMicrosecondsSinceStart` returns the system's clock time, but `GetTimeSinceStart` and its variants return the time since the game started, so I worry that it could cause confusion if someone were to assume all functions named `GetTimeSinceStart` return the same point in time. That's why I chose a very different name that aims to set it further apart from the `GetTimeSinceStart` functions.

Reply:
Maybe something like `GetSystemTimerValue` is more clear about its purpose than `GetElapsedSystemMicroseconds` ?